about.css

```diff
@@ -0,0 +1,426 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  font-family: 'Lato', sans-serif;
+}
+
+.top-header {
+  display: none;
+}
+
+.desktop-nav {
+  display: none;
+}
+
+.logo {
+  display: none;
+}
+
+.menu-icon {
+  background: #d3d3d32d;
+  padding-left: 10px;
+}
+
+.menu-icon img {
+  width: 3em;
+}
+
+.hide {
+  display: none;
+}
+
+.mobile-menu {
+  width: 100%;
+  position: absolute;
+  background-color: black;
+  opacity: 0.8;
+  height: 100vh;
+  top: 0;
+  right: 0;
+  text-align: center;
+}
+
+.mobile-menu ul {
+  margin-top: 70px;
+}
+
+.mobile-menu li {
+  margin-bottom: 20px;
+}
+
+.hero-container a {
+  margin-top: 0.9em;
+  font-weight: 900;
+  font-size: 12px;
+  color: whitesmoke;
+  border-bottom: 1px solid #ec5242;
+}
+
+.mobile-menu li a {
+  color: #fff;
+  position: relative;
+  cursor: pointer;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 25px;
+  padding: 10px;
+  transition: transform 0.5s;
+}
+
+.mobile-menu li a:hover {
+  color: #ec5242;
+}
+
+.cancle-mobile {
+  position: absolute;
+  top: 25px;
+  right: 20px;
+}
+
+#hero {
+  background-image: url(./images/about-hero.jpg);
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  text-align: center;
+}
+
+.hero-container {
+  padding: 20px;
+}
+
+.hero-container h2 {
+  margin-top: 100px;
+  font-weight: 400;
+  font-size: 50px;
+  color: #ec5242;
+}
+
+/* .hero-container h1 {
+  font-size: 50px;
+} */
```

Comment:
- [ OPTIONAL] Please remove all unused code left as comments in your codebase because hey make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍🏾